packages/components/app/styles/components/breadcrumb.scss

```diff
@@ -149,35 +151,39 @@ $hds-breadcrumb-item-visual-horizontal-padding: 4px;
     margin: 0 (-$hds-breadcrumb-item-visual-horizontal-padding); // the horizontal negative margin applied here is for visual balance of the spacing between items
     outline: none; // TODO check with @melanie
     padding: 0;
-    width: 36px; // by design
+    width: $hds-breadcrumb-item-height;
 
     // we apply the focus directly to the element, without using a pseudo-element
     @include hds-focus-ring-basic();
 
     &:hover,
     &.is-hover {
-        background-color: var(--token-color-neutral-background-secondary);
+        border-color: var(--token-color-border-strong);
         color: var(--token-color-foreground-faint);
     }
     &:active,
     &.is-active {
-        background-color: var(--token-color-palette-neutral-200);
-        color: var(--token-color-neutral-foreground-secondary);
+        background-color: var(--token-color-surface-interactive-active);
+        border-color: var(--token-color-border-strong);
+        color: var(--token-color-foreground-primary);
     }
     &:focus,
     &.is-focus {
         background-color: transparent;
+        border: none; // important: we need to completely remove the border, of the inner box-shadow of the focus ring will be drawn inside the border
     }
 }
 
 .hds-breadcrumb__truncation-content {
-    background-color: #FFF;
+    background-color: var(--token-color-surface-primary);
     border-radius: 6px;
     box-shadow: var(--token-surface-high-box-shadow);
     left: -$hds-breadcrumb-item-visual-horizontal-padding;
     margin-top: 4px;
-    padding: 8px 16px;
+    max-width: 200px; // by design
```

Comment:
Issue, blocker: I think you will have to work on this a little bit. It is not handling zoom correctly: 
![CleanShot 2022-03-15 at 08 32 18](https://user-images.githubusercontent.com/4587451/158389037-c1fa3d3f-5c76-4b2d-b62c-21e1d8f6673f.png)

This is a failure of [Reflow (1.4.10)](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) - the intent is to provide  support for people with low vision who need to enlarge text and read it in a single column. When the browser zoom is used to scale content to 400%, it reflows - i.e., it is presented in one column so that scrolling in more than one direction is not necessary.

Reply:
@MelSumner I don't think there is a solution to this (same as below). You will have **exactly** the same issue with the `Dropdown` component. How would you solve it? (I have no idea!)

Reply:
@MelSumner added as "known issue" in https://github.com/hashicorp/design-system/pull/71/commits/fa26a4b3e4684fc9fb8a670d6ad4a8c90f0e2db5